### ISSUE TITLE
More realistic default binary

### ISSUE THREAD

**Issue by bpablo** [phoebe-project/phoebe2#63]
The current default binary parameters are unphysical. I think it would be nice to have a much more reasonable default system. Suggestion for parameters below:

sma@binary = 11.0 Rsol (10 would result in 0.75 Msun which is still at least mildly reasonable)
teff@secondary = 6000
teff@primary = 6000

**Comment by kecnry** [comment]
proposed changes (see e967a1af8dd1fdce32bdc0d8929b39e257920da5):
* period: 10 days (used to be 3)
* sma: 25 solRad (used to be 8)
* teff: 6000 K (used to be 10000)
* gravb_bol: 0.32 (used to be 1.0)
* results in masses of 1.04 solMass
* rpoles are still 1.0 solRad

will need to update all tutorials once this passes tests and is merged

**Comment by kecnry** [comment]
we may now also need to think about a different case for the default overcontact instead of just wrapping an envelope around the default binary

**Comment by aprsa** [comment]
Would it make more sense to make this a closer binary (say, P=1-day), so that distortion effects are pronounced?

**Comment by kecnry** [comment]
I implemented a tighter default system (see 7de0deb38f034879315a6a7bd043b0d89a64614a), but this is causing almost all nosetests to fail - so we'll need to get those running with appropriate tolerances before merging, or fallback on a well-detached system in the nosetests when necessary.
